logstreams/src/test/java/io/zeebe/logstreams/util/AtomixLogStorageRule.java

```diff
@@ -87,12 +89,25 @@ public void appendEntry(
       final long highestPosition,
       final ByteBuffer data,
       final AppendListener listener) {
-    final Indexed<ZeebeEntry> entry =
-        raftLog
-            .writer()
-            .append(
-                new ZeebeEntry(
-                    0, System.currentTimeMillis(), lowestPosition, highestPosition, data));
+    final ZeebeEntry zbEntry =
+        new ZeebeEntry(0, System.currentTimeMillis(), lowestPosition, highestPosition, data);
```

Comment:
Hm this makes me thinking that this setup here is not ideal, since we now actually test this code and not the code in the Raft appender. It is easy that they drift away when we change something here or there. Any idea how we could use the actual code, which we use in production?

Reply:
I agree, I tried to do that but eventually gave up :stuck_out_tongue: it's not ideal but idk of a good wait to improve this without a lot of work

Reply:
Maybe we can use the LeaderAppender somehow

Reply:
Follow up issue?